examples/ms-teams-connector/src/main/java/com/salesforce/einsteinbot/msteams/connector/controller/MSTeamsWebhookListener.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package com.salesforce.einsteinbot.msteams.connector.controller;
+
+import com.microsoft.bot.schema.Activity;
+import com.salesforce.einsteinbot.msteams.connector.service.EinsteinBotService;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * Exposes an endpoint for the outgoing Webhook from MS Teams
+ */
+@RestController
+@RequestMapping("/")
+public class MSTeamsWebhookListener {
+
+  private static final Logger logger = LoggerFactory.getLogger(MSTeamsWebhookListener.class);
+  @Autowired
+  private EinsteinBotService einsteinBotService;
+
+  @PostMapping("/bot")
+  public Activity sendToDo(@RequestBody Activity activity) throws URISyntaxException, IOException
+  {
+    String teamsSessionId = activity.getFrom().getId();
```

Comment:
Will be good to log that a request has been received. Could be a debug log but will be helpful in deugging

Reply:
The request along with sender's name is logged now in debug mode.